### DISCUSSION TITLE
Will parks, squares, rivers etc. appear on OvertureMaps?

### DISCUSSION THREAD

**Discussion by 588chm** [OvertureMaps/data#56] [Discussion]
I was wondering if entities with area-like extent will appear on the OvertureMaps at some point. Like squares on transportation layer. Or parks, forests, rivers in a new theme maybe?

**Comment by jenningsanderson** [comment]
Yes! We are working on building more "context" layers like `land`, `water`, `forest`, `park`, `river`, etc. These will exist under a single "context" theme to be built from the Daylight OpenStreetMap Distribution. 

These layers will most likely be similar to what's available here:  https://registry.opendata.aws/daylight-osm/ and https://daylightmap.org/earth/

**Reply by Cj-Malone** [reply]
That sounds great, now I'm intrigued, are the buildings and transportation themes built from Daylight?
